### ISSUE TITLE
quantile regression

### ISSUE THREAD

**Issue by poojitharamachandra** [dmlc/xgboost#7435]
Hi,

Could you please provide support for quantile regression for uncertainty estimation? (sklearn gradient boosting has one https://scikit-learn.org/stable/auto_examples/ensemble/plot_gradient_boosting_quantile.html)
Is there an alternative already?

**Comment by trivialfis** [comment]
Hi, it's under the roadmap and I have marked https://github.com/dmlc/xgboost/issues/1433 for 2.0 (after 1.6).  Implementing it using only first-order gradient is quite easy, but to get better results we need to export tree leaf index to the objective function, which will take some time.  Stay tuned.  ;-)  Closing as duplicated.
